ERCS/erc-7573.md

```diff
@@ -0,0 +1,288 @@
+---
+eip: 7573
+title: Conditional-upon-Transfer-Decryption for DvP
+description: A Proposal for a Lean and Functional Delivery versus Payment
```

Comment:
The text "A Proposal for a Lean and Functional" doesn't really provide any meaningful information. For the description field to be useful, it needs to explain what makes your proposal useful and different.